packages/cli-config/src/schema.ts

```diff
@@ -160,6 +160,10 @@ export const projectConfig = t
             manifestPath: t.string(),
             packageName: t.string(),
             dependencyConfiguration: t.string(),
+            unstable_reactLegacyComponentNames: t
+              .array()
+              .items(t.string())
+              .default([]),
```

Comment:
**note**: for iOS, we just need these same lines in the iOS project above (Line 149).

The CLI doesn't have to do anything else for iOS, we just need the same key structure.